front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -65,51 +73,61 @@ function useOutputTableData(
     (opPoint) => `${opPoint.name}-${opPoint.ch}`
   );
 
-  const outputTableData = useMemo(
-    () =>
-      suggestedOperationalPoints.map((sugOpPoint, sugOpIndex) => {
-        const opPoint = operationPointsByNameCh[`${sugOpPoint.name}-${sugOpPoint.ch}`];
-        if (!opPoint) {
-          return sugOpPoint;
-        }
-        const nextOpPoint = findNextScheduledOpPoint(
-          operationalPoints,
-          pathStepsWithOpPointIndices,
-          sugOpIndex
-        );
-        const pathStepKey = `${sugOpPoint.uic}-${sugOpPoint.ch}`;
+  const outputTableData = useMemo(() => {
+    const pathStepRows = pathStepsWithPositionOnPath.map((pathStep) => {
+      const schedule = scheduleByAt[pathStep.id];
+      const scheduleData = computeScheduleData(schedule, selectedTrainSchedule.start_time);
+      return { ...pathStep, ...formatScheduleData(scheduleData) };
```

Comment:
We use this combination two times in the file (computeScheduleData then format the result). Maybe we could do the format inside the compute as format is never used without computing first ?

Reply:
Yes I agree, but I would like to do it separately if it's ok for you :)